src/py/pyproject.toml

```diff
@@ -1,2 +1,3 @@
 [build-system]
-requires = ["setuptools>=42", "wheel"]
+requires = ["setuptools>=42"]
+build-backend = "setuptools.build_meta"
```

Comment:
Fixed this up - wheel is not needed (declared by setuptools via PEP 517), and you should always specify a build-backend.